### ISSUE TITLE
livecd-iso-to-disk: kernelargs not going anywhere (EFI)

### ISSUE THREAD

**Issue by e-user** [livecd-tools/livecd-tools#72]
This block of code currently doesn't do anything:
```sh
if [[ -n $kernelargs ]]; then
    sed -i -r "s;\<initrd.\?\.img\>;& ${kernelargs} ;
               s;\<vmlinuz.\?\>;& ${kernelargs} ;" $BOOTCONFIG $BOOTCONFIG_EFI
fi
```

I'm trying to set up a custom netinstall image using `livecd-iso-to-disk --format --efi --ks ks.cfg --label foobar ~/Downloads/Fedora-Workstation-netinst-x86_64-26-1.5.iso /dev/sdc`. ks.cfg gets copied over, but `EFI/BOOT/grub.cfg` doesn't get modified accordingly.

I'm currently using `livecd-tools-24.4-1.fc25.x86_64`.

**Comment by FGrose** [comment]
Pull request #74 should fix this error.

**Comment by Conan-Kudo** [comment]
Fixed in 359ae83caa3dacf1c54dc04da810d2110061c61a.
